x-pack/plugins/spaces/public/views/nav_control/nav_control.tsx

```diff
@@ -59,13 +60,15 @@ module.controller(
     $scope.$parent.$watch('isVisible', function isVisibleWatcher(isVisible: boolean) {
       if (isVisible && !mounted && !pathProvider.isUnauthenticated()) {
         render(
-          <NavControlPopover
-            spacesManager={spacesManager}
-            activeSpace={activeSpace}
-            userProfile={userProfile}
-            anchorPosition={'rightCenter'}
-            buttonClass={SpacesGlobalNavButton}
-          />,
+          <I18nProvider>
```

Comment:
We'll want to add this to the `chromeHeaderNavControlsRegistry.register(...` call below as well

Reply:
Thanks, good catch!